src/components/marquee-wizard/marquee-wizard.component.html

```diff
@@ -48,6 +48,7 @@
                 [attr.aria-labelledby]="id + '-step-' + index + '-label'"
                 [attr.aria-expanded]="step._active"
                 [id]="id + '-step-' + index + '-label'"
+                [attr.tabindex]="step.visited ? 0 : -1"
```

Comment:
I think the actual problem may be with `uxTabbableList`. It should guarantee that the first non-disabled item in the list has tabindex="0". You shouldn't have to set `tabindex` explicitly when using `uxTabbableList`.

https://github.com/UXAspects/UXAspects/blob/master/src/directives/accessibility/tabbable-list/tabbable-list.service.ts#L74

So it might be a change detection issue, or a timing issue where the above line is executed before the first step becomes `visited`.